numpy/ma/tests/test_subclassing.py

```diff
@@ -448,3 +449,12 @@ def test_masked_binary_operations(self):
         assert_(isinstance(np.divide(wm, m2), WrappedArray))
         assert_(isinstance(np.divide(m2, wm), WrappedArray))
         assert_equal(np.divide(m2, wm), np.divide(wm, m2))
+
+    def test_mixins_have_slots(self):
+        mixin = NDArrayOperatorsMixin()
+        # Should raise an error
+        assert_raises(AttributeError, mixin.__setattr__, "not_a_real_attr", 1)
+
+        m = np.ma.masked_array([1, 3, 5], mask=[False, True, False])
+        wm = WrappedArray(m)
+        assert_raises(AttributeError, wm.__setattr__, "not_an_attr", 2)
```

Comment:
I generally prefer `with pytest.raises()` now, but the file uses this, so it seems fine.

Reply:
i first wrote the test with `pytest.raises` then i realized pytest was never imported. So I copied the style from this file.